project_templates/stack_package/example/ups/example.table

```diff
@@ -1,15 +1,9 @@
 # List EUPS dependencies of this package here
 # - Common third-party packages (boost, python, doxygen) and low-level
-#   LSST packages can be assumed to be recursively included by
-#   LSST packages such as utils or daf_base.
+#   LSST packages can be assumed to be recursively included by other
+#   low-level LSST packages.
```

Comment:
I think lines 3 and 4 should be deleted entirely, along with "and low-level" from line 2, since this seem to disagree with lines 5 and 6. At one point the rule that we used for LSST packages was lines 5 and 6, as the outcome of an RFC. I suppose that might have changed or been relaxed a bit for some very low-level LSST packages, but if so, I'm not aware of it.

If things have changed then please at least change "other low-level" to something that makes it clearer that the packages being listed must be higher level than the packages being *implicitly* included. Perhaps just change "other low-level" to "other higher-level". Or just list the few exceptions (surely there aren't many).

Whatever changes you make here, please also make them in all other appropriate .table files in this package.